### PR TITLE
Phase 1 - 1.1 Mock implementation

### PR DESCRIPTION
**Pull request by kanghowoo**
Phase 1 - 1.1 Mock implementation 
Requirement :
Let’s create controller and service/storage interface for WRITE post
And for service/storage implementations, please create mock implementation first.

### CODE REVIEW

File: src/main/java/com/mide/gangsaeng/board/BoardRepository.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package com.mide.gangsaeng.board;
+
+import org.springframework.stereotype.Repository;
+
+@Repository
+public class BoardRepository {
```
Comment (TK-one) [inline src/main/java/com/mide/gangsaeng/board/BoardRepository.java:6]: Create interface first like service

File: src/main/java/com/mide/gangsaeng/board/BoardController.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+@RequestMapping("/board")
+public class BoardController {
+
+    private final BoardService boardService;
+
+    @Autowired
+    public BoardController(BoardService boardService) {this.boardService = boardService;}
+
+    @PostMapping
+    public ResponseEntity<String> upload() {
```
Comment (TK-one) [inline src/main/java/com/mide/gangsaeng/board/BoardController.java:20]: change method name to `write` or use same method name in service layer

File: src/main/java/com/mide/gangsaeng/board/BoardController.java [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/board")
+public class BoardController {
+
+    private final BoardService boardService;
+
+    @Autowired
```
Comment (TK-one) [inline src/main/java/com/mide/gangsaeng/board/BoardController.java:16]: Can we remove this `@Autowired` annotation?

Comment (kanghowoo) [reply src/main/java/com/mide/gangsaeng/board/BoardController.java:15]: Yes, when a class has only one constructor or Spring 4.3 version onwards, but in this project, i will write `@Autowired` for explicitness.

**Review by TK-one — APPROVED** [review]

File: src/main/java/com/mide/gangsaeng/board/BoardController.java (hunk 3 above)
Comment (kanghowoo) [reply src/main/java/com/mide/gangsaeng/board/BoardController.java:15]: Sorry, upong searching, when spring 4.3 version onwards `and` a class has only one constructor, we can remove `@Autowired` annotation.
You can check this contents in spring docs. 
[https://docs.spring.io/spring-framework/reference/core/beans/annotation-config/autowired.html](url)

**Pull request merged**